File: src/musicgo/src/shared/classes/ClaimViewModel.ts
import { SongViewModel } from './SongViewModel';
import { StateViewModel } from './StateViewModel';
import { UserViewModel } from './UserViewModel';

export class ClaimViewModel{
    public Description: string = "";

    public SongClaimed : SongViewModel = new SongViewModel();

    public State : StateViewModel = new StateViewModel();

    public User : UserViewModel = new UserViewModel();

    public Date : Date = new Date();
}

File: ViewModels/ViewModels/ClaimViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ViewModels.ViewModels
{
    public class ClaimViewModel : BaseViewModel
    {
        public string Description { get; set; }

        public SongViewModel SongClaimed { get; set; }

        public StateViewModel State { get; set; }

        public UserViewModel User { get; set; }

        public DateTime Date { get; set; }
    }
}
